packages/block-library/src/missing/edit.native.js

```diff
@@ -120,6 +127,26 @@ export class UnsupportedBlockEdit extends Component {
 		}
 	}
 
+	getSheetTitle( blockTitle ) {
+		if ( CUSTOM_UNSUPPORTED_BLOCK_MESSAGE[ blockTitle ] ) {
```

Comment:
Maybe instead of using the block's title, we could use the name (i.e. `core/gallery`) as it might be closer to being an identifier and hence not subject to change, wdyt?

Reply:
Good idea, I updated it 👍